packages/webapp/src/hooks/useGetMenuItems.jsx

```diff
@@ -50,6 +55,29 @@ export const useGetMenuItems = () => {
         path: '/crop_catalogue',
         key: 'crops',
       },
+      {
+        label: t('MENU.ANIMALS'),
+        icon: <AnimalsIcon />,
+        path: ANIMALS_HOME_URL,
```

Comment:
This is the main line I felt unsure about as for finances it uses the non-existent route "/finances". So should I be using "/animals" despite the fact it would route to home if it ever rendered

Reply:
Right, it should be the non-existent route `/animals` (same as finances) -- there is no way to render it (follow it) from the menu anyway, it is just for doing the path matching for the menu highlights.

The way it is now specifying `animal/inventory` recreates the menu highlight bug we made in finances that was fixed in this PR: #3086. I don't know if it matters but in that PR @antsgar kept the `HOME_URL` constant as `/transactions` (here it would be `/inventory`) and  made the plain URL its own constant; probably we would want to keep that for consistency.

Reply:
k updated